src/QirTools/pyqir/setup.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+import sys
+
+from setuptools import setup
+from setuptools_rust import Binding, RustExtension
+
+setup(
+    name="pyqir",
+    version="0.0.1",
```

Comment:
We should ideally pick up the version number from CI, similar to what's done for other QDK Python packages.

Reply:
Handled offline and in later commits.